quickstarts/01-hello-world/README.md

```diff
@@ -402,6 +430,8 @@ expected_stdout_lines:
 output_match_mode: substring
 -->
 ```bash
+source .venv/bin/activate
```

Comment:
hmmm we might need to do a lookup cmd on the venv thats active bc i typically name my venv now and almost never use .venv personally...

Reply:
as I moved the mechanical markdown step to be above (https://github.com/dapr/dapr-agents/pull/214/files#diff-cf7137d2a671ec12fdf16560e2e48a9f6f250acbaefdb80282ecf47d94ea8ca6R25) the uv setup it should take care of using/setting up the proper environment.